### PR TITLE
Rm static testnets

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
Static testnets are dead.  ~Long live static testnets.~

# Changes
Delete or rename references to the `nnsdapp` testnet in many, but not all places.  In particular, changes to the release SOP require special attention so they deserve to be be in a separate PR.

Note: Each set of changes is in a separate commit, so you can check the changes as atomic units.  Any that turn out to be complicated can be spun out into separate PRs.

# Tests
See CI

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -111,34 +111,16 @@ TODO: Document how to create accounts in the nns-dapp.
 
 ## Development
 
-Development relies on the presence of a testnet that is setup with the II, governance, ledger, and cycle minting canisters. Fully local development is unfortunately not yet supported and the tools for setting up a testnet are not yet available publicly. It is on the roadmap to make these tools available publicly for developers.
+Development relies on the presence of a testnet that is setup with the II, governance, ledger, and cycle minting canisters.  This is provided by [`dfx nns install`](https://internetcomputer.org/docs/current/references/cli-reference/dfx-nns), both in automated testing and for local development.  Please note, however, that it takes some work to configure the NNS canisters.  If you would like a test environment with all the NNS canisters configured suitably for use with the nns-dapp, please consider using the [snsdemo tools.](https://github.com/dfinity/snsdemo):
```
Comment (lmuntaner) [inline BUILD.md:114]: "Development relies on the presence of a testnet", maybe change "testnet" for "environment"?

File: dfx.json
Comment (lmuntaner) [inline dfx.json:1]: Why not removing all the small, large, benchmark ?

File: BUILD.md (hunk 1 above)
Comment (bitdivine) [reply BUILD.md]: Done

File: dfx.json
Comment (bitdivine) [reply dfx.json:1]: I was just warily focused on the one testnet but of course they can all go.  Done.

**Comment by bitdivine** [comment]
Formatting ...

**Review by lmuntaner — APPROVED** [review]
Nice!!

**Pull request merged**